inception-project/inception
# #2879 - Improve multi-value features

**What's in the PR**
- Recommenders: When a recommendation is on a multi-value feature, do not hide overlapping suggestions (same as when stacking is enabled) - unless label matches existing annotation

**How to test manually**
* Create span layer with multi-value string feature
* Create string matching recommender for that feature
* Annotate a word in a text with two labels on the same annotation
* Check suggestions projected onto other instances of the same word

**Automatic testing**

**Documentation**
